net/arp-scan/Makefile

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright (C) 2006-2016 OpenWrt.org
```

Comment:
I am not a lawyer, but if this file is only introduced this year, then just `2016` should be used.